packages/shared/components/atoms/pills/DeveloperIndicatorPill.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="typescript">
+    import { Pill } from 'shared/components'
+
+    export let label: string = ''
+</script>
+
+<Pill
+    data={label}
+    smaller
+    textColor="gray-600"
+    darkTextColor="white"
+    backgroundColor="gray-100"
+    darkBackgroundColor="gray-700"
+    classes="uppercase dark:bg-opacity-20 rounded-full px-3"
+/>
```

Comment:
No need to pass in the label here as we making a consistent component for the DEV indicator only
```suggestion
<script lang="typescript">
    import { localize } from '@core/i18n'
    import { Pill } from 'shared/components'
</script>

<Pill
    data={localize('general.dev')}
    smaller
    textColor="gray-600"
    darkTextColor="white"
    backgroundColor="gray-100"
    darkBackgroundColor="gray-700"
    classes="uppercase dark:bg-opacity-20 rounded-full px-3"
/>
```